### DISCUSSION TITLE
Hide Cover Art Band Setting Ignored

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1772] [Fixed Issues]
When I access NAS folders in the Library, the Hide Cover Art Band Option does not seem to work.

**Comment by R101G** [comment]
@rern
This problem only shows on my second system, which has a local display and no SD or USB files, only the NAS mount. The main system, which has USB files plus NAS, works as it should.

**Reply by rern** [reply]
Please try the UPDATE.

**Reply by R101G** [reply]
@rern 
That seems to have fixed it. Thanks.
